### DISCUSSION TITLE
How to clear devices

### DISCUSSION THREAD

**Discussion by ZhangXuedan** [codeceptjs/CodeceptJS#3616] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/20330132/224198578-36de1964-48a0-4cae-aa52-cf1a5b100016.png)
Here is my code,When I run, the first session window and the second session window become the same size, but I only set the first session window as iPhone 13 Pro
![image](https://user-images.githubusercontent.com/20330132/224198838-ea2435b1-b9ee-415b-8142-26bca4a4f782.png)
Please help me if you have a good method

**Comment by adudyak** [comment, marked as answer]
I see `;` is missing here and there, I would add it just in case.
In case you don't specify config for session, it should take defaults from config file. Are there any defaults?
Try to pass `{browser: 'chromium'}` as 2nd parameter to your second session.

**Reply by ZhangXuedan** [reply]
Thanks for your answer, it's very useful to me
